backend/battles/admin.py

```diff
@@ -7,5 +7,9 @@ class TeamAdmin(admin.ModelAdmin):
     list_display = ("trainer", "battle")
```

Comment:
![image](https://user-images.githubusercontent.com/23136832/78128335-16c97900-73ec-11ea-975a-5c79bb5ef96a.png)

Can I select the same pokemon as my opponent? I remember not being allowed on the specs.

Reply:
I suppose so, there were no restriction of duplicate teams in the project specs.